File: zerojudge/a015/main.py
while True:
    try:
        inp = input().split(" ")
        row, col = int(inp[0]), int(inp[1])

        table = []
        for i in range(row):
            temp = input().split(" ")
            for n in range(col):
                temp[n] = temp[n]
            table.append(temp)

        for i in range(col):
            for j in range(row):
                print(table[j][i], end=" ")
            print("")

    except EOFError:
        break

File: zerojudge/a015/main.cpp
#include<iostream>
#include<vector>
using namespace std;
int main(){
    int row, col;
    while (cin>>row>>col){
        vector<vector<int> > table;
        vector<int> temp_vec;
        int temp;
        for (int i=0; i<row; i++){
            for (int j=0; j<col; j++){
                cin>>temp;
                temp_vec.push_back(temp);
            }
            table.push_back(temp_vec);
        }

        for (int i=0; i<col; i++){
            for (int j=0; j<row; j++){
                cout<<table[j][i]<<" ";
            }
            cout<<"\n";
        }

    }
    return 0;
}